frontend/src/api/RoomzMediaServerClient.js

```diff
@@ -0,0 +1,81 @@
+
+import io from 'socket.io-client';
+// Connect to RMS
+const rmsClientSocket = io("http://localhost:5000", {
+  autoConnect: false,
+  reconnection: true,
+  reconnectionAttempts: 2,
+  timeout: 10000 // timeout for each reconnection attempt
+});
+
+async function askToConnect() {
+  try {
+    await rmsClientSocket.connect()
+    return true
+  } catch (e) {
+    console.error("Failed to create socket connection: %s", e)
+    return false
+  }
+}
+
+const events = {
+  JOIN_MEDIA_ROOM: 'JoinMediaRoom',
+  RECEIVE_MEDIA_FROM: 'ReceiveMediaFrom',
+  EXISTING_MEDIA_ROOMIEZ: 'ExistingMediaRoomiez',
+  NEW_MEDIA_ROOMY_ARRIVED: 'NewMediaRoomyArrived',
+  RECEIVE_MEDIA_ANSWER: 'ReceiveMediaAnswer',
+}
```

Comment:
Defining constants like this is good instead of having the strings hardcoded in each function.